File: RepoTransBench/source_projects/Java/JamesZBL_java_design_patterns/composite/src/test/java/me/zbl/composite/EnglishSentencePublicTest.java
package me.zbl.composite;

import org.junit.jupiter.api.Test;

import java.io.ByteArrayOutputStream;
import java.io.PrintStream;
import java.util.Arrays;

import static org.junit.jupiter.api.Assertions.*;

class EnglishSentencePublicTest {

    @Test
    void testSentenceCompositionPublic() {
        EnglishWord word1 = new EnglishWord(Arrays.asList(new Character('T'), new Character('e'), new Character('s'), new Character('t')));
        EnglishWord word2 = new EnglishWord(Arrays.asList(new Character('P'), new Character('u'), new Character('b'), new Character('l'), new Character('i'), new Character('c')));
        EnglishSentence s = new EnglishSentence(Arrays.asList(word1, word2));
        assertEquals(2, s.count());
        ByteArrayOutputStream baos = new ByteArrayOutputStream();
        PrintStream old = System.out;
        System.setOut(new PrintStream(baos));
        s.print();
        System.setOut(old);
        assertTrue(baos.toString().contains("."));
    }
}

File: RepoTransBench/target_projects/Java/C#/JamesZBL_java_design_patterns/public_tests/EnglishSentencePublicTest.cs
using System;
using System.Collections.Generic;
using System.IO;
using Xunit;

namespace PublicTests
{
    public class EnglishSentencePublicTest
    {
        [Fact]
        public void TestSentenceCompositionPublic()
        {
            var word1 = new EnglishWord(new List<Character> {
                new Character('T'), new Character('e'), new Character('s'), new Character('t')
            });
            var word2 = new EnglishWord(new List<Character> {
                new Character('P'), new Character('u'), new Character('b'), new Character('l'), new Character('i'), new Character('c')
            });
            var s = new EnglishSentence(new List<EnglishWord> { word1, word2 });
            Assert.Equal(2, s.Count());
            var baos = new StringWriter();
            var oldOut = Console.Out;
            Console.SetOut(baos);
            s.Print();
            Console.SetOut(oldOut);
            Assert.Contains(".", baos.ToString());
        }
    }
}